htdocs/core/class/html.form.class.php

```diff
@@ -8030,6 +8030,7 @@ public function selectForFormsList($objecttmp, $htmlname, $preselectedvalue, $sh
 
 		// Search data
 		$sql = "SELECT t.rowid, " . $fieldstoshow . " FROM " . $this->db->prefix() . $objecttmp->table_element . " as t";
+		$sql .= " LEFT JOIN " . $this->db->prefix() . $objecttmp->table_element . "_extrafields as e ON t.rowid=e.fk_object";
```

Comment:
Not that some object has no extrafields so the table does not exists.
You may add a test on if ($objecttmp->isextrafieldmanaged) before adding this left join...

Reply:
True, thank you!